### ISSUE TITLE
Keyboard trap occurs

### ISSUE THREAD

**Issue by dmoxin** [highcharts/highcharts#17374]
#### Expected behaviour
With the accessibility module added, I want to be able to use the tab button to navigate in and through the focusable elements within a Highchart Chart.  When I continue tabbing through the chart, the focus should access other elements on the web page that appear after the chart in the DOM.

#### Actual behaviour
After manipulating/navigating within the chart, I sometimes become caught in a loop.  I can shift + tab to back out of the chart, but I cannot tab through the chart to other elements on the page.

#### Live demo with steps to reproduce

https://www.richmondfed.org/offsite/darren/chart_local
There does not seem to be a sure-fire way to consistently recreate the issue.  I usually need to do some combination of opening and closing the hamburger menu, then toggling items in the chart legend on and off.

#### Product version

Highcharts 9.0.1

#### Affected browser(s)
All

**Comment by dmoxin** [comment]
Hi @oysteinmoseng ...I will need to check with my information security people to make sure I'm allowed to post the code.  There's nothing particularly special about it, its just that we have very strict security guidelines.  I circle back shortly.  Thanks.

**Comment by dmoxin** [comment]
Hey @oysteinmoseng... any new information on this?

**Comment by oysteinmoseng** [comment]
Hi @dmoxin, sorry, I did not get notified about your post edit with the question!

The advantage of getting it from codepen/jsFiddle is that we can see the options you pass to the chart to produce the result.  This makes it a lot easier to reproduce on our side. We can try to scrape and hack something together from the chart result on the page, but it's cumbersome and from experience often error-prone. It would be great if you could get the OK from your security team to share it. If sharing it publicly is problematic, feel free to send us a support ticket.

**Comment by dmoxin** [comment]
Hey @oysteinmoseng I cut a page down to the essentials and posted it on jsfidde.  Let me know if you need things set up a different way.

https://jsfiddle.net/7zmdwujb/

I'm able to consistently recreate the issue with the following steps:

1. Bring focus to the 'Focusable Element' link.
2. Hit tab to put focus on the chart lines, then use left/right arrow keys to move around a bit
3. Hit tab to place focus on one of the items in the legend then hit enter to toggle a line off/on
4. Tab up to the hamburger/menu icon
5. Hit enter to open the export menu it, move up/down with the arrow keys
6. Hit tab to move away from the menu icon.  

Now as I continue to press tab, the focus is caught in a loop between the lines, legend, and export menu.

Thanks for the continued help.

**Comment by oysteinmoseng** [comment]
@dmoxin Excellent, thank you for the demo. I am able to reproduce it on my side and will take a look shortly.

**Comment by oysteinmoseng** [comment]
@dmoxin This no longer seems to be an issue with later Highcharts versions, so closing this for now. Feel free to comment to reopen if needed. See https://jsfiddle.net/o6c9svgd/
